AsyncDisplayKit/TextKit/ASTextKitRenderer.mm

```diff
@@ -82,7 +82,20 @@ - (void)_calculateSize
   CGRect constrainedRect = {CGPointZero, _constrainedSize};
   __block CGRect boundingRect;
   [_context performBlockWithLockedTextKitComponents:^(NSLayoutManager *layoutManager, NSTextStorage *textStorage, NSTextContainer *textContainer) {
+    NSParagraphStyle *paragraphStyle = [textStorage attributesAtIndex:[layoutManager characterIndexForGlyphAtIndex:0]
```

Comment:
Good point and you caught my laziness :) modified it to only take the paragraph style of the last visible glyph and it still fixed it.